test/parallel/test-fs-open-flags.js

```diff
@@ -22,6 +23,8 @@ assert.strictEqual(stringToFlags('w+'), O_TRUNC | O_CREAT | O_RDWR);
 assert.strictEqual(stringToFlags('a'), O_APPEND | O_CREAT | O_WRONLY);
 assert.strictEqual(stringToFlags('a+'), O_APPEND | O_CREAT | O_RDWR);
 
+assert.strictEqual(stringToFlags('rs+'), O_RDWR | O_SYNC);
```

Comment:
please put after `r+`, so that the asserts are in documentation order, making it much easier to check that all documented flags are tested.

Also, I notice that your test for `sr+`, but that is *not* documented, but neither are `xw`, etc, which are also tested here.

I'll document the combinations, unless you are interested in PRing a doc update as well?